scripts/globals/status.lua

```diff
@@ -1021,6 +1021,7 @@ dsp.mod =
     FASTCAST                        = 170,
     UFASTCAST                       = 407,
     CURE_CAST_TIME                  = 519,
+    ELEMENTAL_CELERITY 	            = 901, -- Quickens Elemental Magic Casting
```

Comment:
![image](https://user-images.githubusercontent.com/6871475/39437780-db882c74-4c6f-11e8-907e-06a29cf168e9.png)
Looks like one still snuck past you

Reply:
Deleting the whole fucking whitespace and re-tabbing. Piece of shit editor.

Reply:
I don't have any changes to the file. You sure?

Reply:
Nvm. I'm an idiot.

Reply:
its what github showed for me, and that was the full (all commits) view after your last push.